src/felles/komponenter/rekbis-kort/RekBisKort.tsx

```diff
@@ -0,0 +1,23 @@
+import { Panel } from '@navikt/ds-react';
+import classNames from 'classnames';
+import * as React from 'react';
+import css from './RekBisKort.module.css';
+
+export interface IRekBisKort {
```

Comment:
Mulig det bare er meg, men jeg leser dette navnet som `Kort Rekbis`. Er det bedre å bare kalle det for `Rekrutteringsbistandkort`? Hva er et kort, egentlig? Kunne det hett noe mer beskrivende, f.eks. `Layoutkort`, `Sidekort`, `Innholdskort` elns.?